src/pages/About/index.jsx

```diff
@@ -1,14 +1,45 @@
+import React, { Component } from 'react';
+
 import * as Style from './styles';
 
 import Header from '../../components/layout/header/Header';
 
-const About = () => (
-  <>
-  <Header />
-  <Style.MainContainer>
-    <Style.TitleContent>This is about page</Style.TitleContent>
-  </Style.MainContainer>
-  </>
-);
+class About extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
cái super này để làm gì vậy em, nếu không dùng nó thì có bị sao không

Reply:
super này để gọi lại constructor của class cha (component). MÌnh sẽ không thể sử dụng this ở đây nếu chưa gọi tới contructor của class cha.